src/main/resources/assets/mwc/lang/en_us.lang

```diff
@@ -1,5 +1,12 @@
 # HUD
 
+#ItemGroups
+itemGroup.resourcestab=Resources
```

Comment:
Also use capitalization for the love of god, so it should be `resourcesTab`

Also try giving a more descriptive name players don't play with only MWC so maybe MWC Resources?